src/Components/Classes/index.jsx

```diff
@@ -57,12 +51,12 @@ function Projects() {
   return (
     <section className={styles.container}>
       <h2>Class List</h2>
+      {pending && <Loader />}
       {classes.length !== 0 ? (
         <ClassList
           classes={classes && classes}
           getById={getById}
           selectedClass={selectedClass}
-          setRenderData={setRenderData}
         ></ClassList>
       ) : (
         'There are not classes yet. Add new ones!!'
```

Comment:
This message should only appear once when no classes are retrieved, and not simultaneously with the spinner.